Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView/Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView.csproj

```diff
@@ -0,0 +1,107 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{81B7758C-233E-489A-8F6D-0EC25660F46E}</ProjectGuid>
+    <OutputType>WinExe</OutputType>
+    <RootNamespace>Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView</RootNamespace>
+    <AssemblyName>Microsoft.Toolkit.Win32.Samples.WPF.WebView.HostingWinFormsWebView</AssemblyName>
+    <TargetFrameworkVersion>v4.7.2</TargetFrameworkVersion>
```

Comment:
@nmetulev @normesta @mcleanbyron  We have at least 4.7 for the main controls package due to issues with high DPI modes that are fixed in 4.7. 4.7.2 contains additional fixes for DPI, but the feedback so far is this might be a non-starter if we are to force existing applications to go from 4.5 ish to 4.7+

Reply:
As long as you document the issues, I think it's ok for the developer to make that choice.